src/components/Modals/AddContactModal.jsx

```diff
@@ -85,14 +86,16 @@ const AddContactModal = ({ addContact, showAddContactModal, setShowAddContactMod
     }
   };
 
+  const formTitle = 'Add Contact';
+
   return (
     <Dialog
       open={showAddContactModal}
       aria-labelledby="dialog-title"
       onClose={() => setShowAddContactModal(false)}
     >
-      <FormSection title="Add Contact">
-        <form onSubmit={handleAddContact} autoComplete="off">
+      <FormSection title={formTitle}>
+        <form aria-label={formTitle} onSubmit={handleAddContact} autoComplete="off">
```

Comment:
I believe `aria-label` should be used when there is no content in the DOM that can be used to label this element.

So, here I believe we should use `aria-labelledby` and reference the `<FormSection>` element which labels this form via it's title.